default/src/components/transfer-asa.js

```diff
@@ -0,0 +1,68 @@
+/* global AlgoSigner */
+import algosdk from 'algosdk';
+
+import { LEDGER } from '../algosigner.config';
+
+const LAST_ROUND = 'last-round';
+const CONFIRMED_ROUND = 'confirmed-round';
+
+const algodClient = new algosdk.Algodv2(
+  'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+  'http://localhost',
+  4001
+);
```

Comment:
we should use `AlgoSigner.algod` instead of creating a custom client.

Reply:
okay